contrib/aubio/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "aubio"
+pkgver = "0.4.9"
+pkgrel = 0
+build_style = "waf"
+hostmakedepends = ["python", "pkgconf", "doxygen", "python-sphinx", "txt2man"]
+makedepends = [
+    "ffmpeg-devel",
+    "libsamplerate-devel",
+    "libsndfile-devel",
+    "pipewire-jack-devel",
+]
+pkgdesc = "Library for audio and music analysis"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "GPL-3.0-or-later"
+url = "https://aubio.org"
+# bundled waf uses deprecated python modules
+_waf_ver = "2.0.25"
+source = [
+    f"{url}/pub/aubio-{pkgver}.tar.bz2",
+    f"!https://waf.io/waf-{_waf_ver}",
+]
+sha256 = [
+    "d48282ae4dab83b3dc94c16cf011bcb63835c1c02b515490e1883049c3d1f3da",
+    "21199cd220ccf60434133e1fd2ab8c8e5217c3799199c82722543970dc8e38d5",
+]
+# tests run in do_build
+options = ["!check"]
+
+# don't use deprecated ffmpeg functions
+tool_flags = {"CFLAGS": ["-DFF_API_LAVF_AVCTX"]}
```

Comment:
why not? it shouldn't matter for the build itself

Reply:
ah i see

apply https://github.com/aubio/aubio/commit/8a05420e5dd8c7b8b2447f82dc919765876511b3 instead so it doesn't get lost in the future to be removed :)